src/radix_connect/wallet_account.rs

```diff
@@ -0,0 +1,27 @@
+use crate::prelude::*;
+
+#[derive(Debug, Serialize, PartialEq, Clone, uniffi::Record)]
+#[serde(rename_all = "camelCase")]
+pub struct WalletInteractionWalletAccount {
+    pub address: AccountAddress,
+    pub label: String,
+    pub appearance_id: AppearanceID,
+}
+
+impl HasSampleValues for WalletInteractionWalletAccount {
```

Comment:
prefer implementing this constructor:

```rust
impl WalletInteractionWalletAccount {
    pub fn new(address: impl Into< AccountAddress>, label: impl AsRef<str>, appearance_id: impl Into< AppearanceID >) -> Self {
        Self { address: address.into(), label: label.as_ref().to_owned(), appearance_id: appearance_id.into() }
    }
}
```

that way the Sample value impl - and all test code becomes much simpler! e.g.:

```rust
  Self {
            address: AccountAddress::sample_other(),
            label: "sample2",
            appearance_id: AppearanceID::sample_other(),
        }
```
and you can even pass addresses by string if you want :)